tests/api/v1/datadog/cassettes/TestUsageGetBillableSummaryErrors/400_Bad_Request.yaml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Since the 400 Bad Request part of the "TestUsageGetBillableSummaryErrors` test is commented out, we shouldn't need this cassette anymore